datadog_checks_downloader/datadog_checks/downloader/cli.py

```diff
@@ -23,6 +24,34 @@ def __is_canonical(version):
     return re.match(P, version) is not None
 
 
+def __find_shipped_integrations():
+    root = os.path.dirname(os.path.abspath(__file__))
+    filename = 'requirements-agent-release.txt'
+
+    integrations = set()
+
+    while True:
+        file_path = os.path.join(root, filename)
+        if os.path.isfile(file_path):
+            break
+
+        new_root = os.path.dirname(root)
+        if new_root == root:
+            return integrations
+
+        root = new_root
```

Comment:
I think we should just indicate the exact path of `requirements-agent-release.txt` instead of walking up, the code will be cleaner IMHO.

Yes, we might need to update path if the location changes, but that's fine I think.

Reply:
We do this in the Agent too because the path differs between OSes

Reply:
Ok I see, but not really fan of that :)

Maybe we can make the "find path by walking up" a util method ?

Reply:
I agree. To me, it's still not clear what this code does. Should the `integrations` set ever be returned empty?